File: Java/ESAMI/Esame11Aprile2019/Nutrizionista.java
public class Nutrizionista extends Dipendente {

    private int telefono, appuntamentiSettimanali;
    private boolean medico;

    public Nutrizionista(int codice, String nomeDipendente, double costoOrario, int telefono, boolean medico, int appuntamentiSettimanali) {
        super(codice, nomeDipendente, costoOrario);
        this.telefono = telefono;
        this.medico = medico;
        this.appuntamentiSettimanali = appuntamentiSettimanali;
    }

    public String toString() {
        return nomeDipendente + "\t" + codice + "\t" + "nutri" + "\t" + "-" + "\t" + "-" + "\t" + "-" + "\t" + telefono + "\t" + medico + "\t" + appuntamentiSettimanali + "\t" + costoOrario;
    }
    
}

File: SecondoAnno/Programmazione_II/Python/esame_11_04_2019/nutrizionista.py
from dipendente import Dipendente

class Nutrizionista(Dipendente):
    telefono: int
    medico: bool 
    appuntamenti_settimanali: int 

    def __init__(self, codice: int, nome: str, costo_orario: float, telefono: int, medico: bool, appuntamenti_settimanali: int) -> None:
        super().__init__(codice, nome, costo_orario)
        self.telefono = telefono
        self.medico = medico
        self.appuntamenti_settimanali = appuntamenti_settimanali

    def __str__(self) -> str:
        return super().__str__() + " nutri , - , - , " + str(self.telefono) + " , " + str(self.medico) + " , " + str(self.appuntamenti_settimanali) + " , " + str(self.costo_orario)